lib/presentation/organisms/timer_organism.dart

```diff
@@ -24,6 +26,10 @@ class _TimerOrganismState extends State<TimerOrganism>
   @override
   void initState() {
     super.initState();
+    lockScreen = _prefs.getBool("isSilent") ?? lockScreen;
```

Comment:
You are getting the wrong value.
isSilent to a lockScreen var?

We have both
![image](https://github.com/guyluz11/infinite_horizons/assets/9304740/84e5f18e-a769-4f22-96ce-aa5281d37db8)

Reply:
My Bad.